docs/quickstart.md

```diff
@@ -0,0 +1,139 @@
+
+## Prerequisites
+
+- Install [Docker] and [kind]
+- Install [kubectl]
+- Install [clusterctl]
+- Install [go]  # installation of the Go package `envsubst` is required to enable the expansion of variables specified in CSPO and CSO manifests.
+
+## Initialize the management cluster
+
+Create the kind cluster:
+
+```bash
+kind create cluster
+```
+
+Transform the Kubernetes cluster into a management cluster by using `clusterctl init` and bootstrap it with CAPI and Cluster API Provider OpenStack ([CAPO]) components:
+
+```bash
+# Enable Cluster Class CAPI experimental feature
+export CLUSTER_TOPOLOGY=true
+
+# Install CAPI and CAPO components
+clusterctl init --infrastructure openstack
+```
+
+### Create a secret for OpenStack access
+
+To enable communication between the CSPO and the Cluster API Provider for OpenStack (CAPO) with the OpenStack API, it is necessary to generate a secret containing the access data (clouds.yaml).
+Ensure that this secret is located in the identical namespace as the other Custom Resources.
+
+> [!NOTE]  
+> The default value of `cloudName` is configured as `openstack`. This setting can be overridden by including the `cloudName` key in the secret. Also, be aware that the name of the secret is expected to be `openstack` unless it is not set differently in OpenStackClusterStackReleaseTemplate in `identityRef.name` field.
+
+```bash
+kubectl create secret generic openstack --from-file=clouds.yaml=path/to/clouds.yaml
+
+# Patch the created secrets so they are automatically moved to the target cluster later.
+
+kubectl patch secret openstack -p '{"metadata":{"labels":{"clusterctl.cluster.x-k8s.io/move":""}}}'
+```
+
+### CSO and CSPO variables preparation
+
+The CSO and CSPO must be directed to the Cluster Stacks repository housing releases for the OpenStack provider.
+Modify and export the following environment variables if you wish to redirect CSO and CSPO to an alternative Git repository
+
+Be aware that GitHub enforces limitations on the number of API requests per unit of time. To overcome this,
+it is recommended to configure a personal access token for authenticated calls. This will significantly increase the rate limit for GitHub API requests.
+
+```bash
+export GIT_PROVIDER_B64=Z2l0aHVi  # github
+export GIT_ORG_NAME_B64=U292ZXJlaWduQ2xvdWRTdGFjaw==  # SovereignCloudStack
+export GIT_REPOSITORY_NAME_B64=Y2x1c3Rlci1zdGFja3M=  # cluster-stacks
+export GIT_ACCESS_TOKEN_B64=<my-github-access-token>
+```
+
+### CSO and CSPO deployment
+
+Install the [envsubst] Go package. It is required to enable the expansion of variables specified in CSPO and CSO manifests.
+
+```bash
+GOBIN=/tmp go install github.com/drone/envsubst/v2/cmd/envsubst@latest
+```
+
+Get the latest CSO release version and apply CSO manifests to the management cluster.
+
+```bash
```

Comment:
The latest can be retrieved by `https://github.com/SovereignCloudStack/cluster-stack-operator/releases/latest/download/cso-infrastructure-components.yaml`

We do not need `jq`(missing from prerequisites) then

Reply:
good point, but let's prefer explicit before implicit, and I would rather prefer to add `jq` to the prerequisites

Reply:
Ok, I agree

Reply:
fixed: c380e5fa26ad5d4725148dfaf08bed0f842afcc8